TurkishFilter/sections/general_extensions.txt

```diff
@@ -284,9 +284,10 @@ kofdizi.com#%#AG_onLoad(function() { jQuery(function() { jQuery('body').unbind('
 t24.com.tr#%#AG_onLoad(function() { $('a[href^="http://t24.com.tr/haber/advertorial"]').closest('li[itemprop$="itemListElement"]').remove(); $('a[href^="http://t24.com.tr/images/counter.php?ad"]').remove(); });
 ! https://forum.adguard.com/index.php?threads/21052/
 finikeakdeniz.com.tr#%#AG_onLoad(function() { if (window.location.href.indexOf("/reklam.asp") != -1) { window.location.href = "/" }; });
-!
 ! http://forum.adguard.com/showthread.php?7046
 mackolik.com,sahadan.com#%#setTimeout ("HideFloatAdBanner()", 1000);
+! https://github.com/AdguardTeam/AdguardFilters/issues/65929#issuecomment-714316741
+r10.net#%#document.cookie = 'r10LeftHide=1'
```

Comment:
Using scriptlet here instead of JS causes first load to still show the left-side banner. Hence using JS here.